.github/workflows/update.yml

```diff
@@ -0,0 +1,42 @@
+name: Update from Releases
+
+on:
+  push:
+  schedule:
+     # At the end of every day
+     - cron: 0 0 * * *
+
+jobs:
+
+  Update:
+
+    runs-on: ubuntu-16.04
+
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Install Python Modules
+      run: |
+        python --version
+        pip install PyGitHub==1.45
+
+    - name: Update
+      id: update
+      run: |
+        ./update.py --set-outputs 1
+
+    - name: Create Pull Request
```

Comment:
Is this relying on `create-pull-request` to be a no-op if there are no changes?

Reply:
Correct, and that is the documented behaviour : 

> If there are no changes (i.e. no diff exists with the checked-out base branch), no pull request will be created and the action exits silently.